src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowSnykScanListenerLS.kt

```diff
@@ -155,45 +162,102 @@ class SnykToolWindowSnykScanListenerLS(
             ossResultsCount = ossResultsCount,
             iacResultsCount = iacResultsCount,
             containerResultsCount = containerResultsCount,
-            addHMLPostfix = rootNodePostFix
+            addHMLPostfix = rootNodePostFix,
         )
 
         snykToolWindowPanel.smartReloadRootNode(
             rootNode,
             userObjectsForExpandedNodes,
-            selectedNodeUserObject
+            selectedNodeUserObject,
         )
     }
 
-    private fun displayResultsForRootTreeNode(
+    @Suppress("RedundantVisibilityModifierRule")
+    fun addInfoTreeNodes(
         rootNode: DefaultMutableTreeNode,
-        issues: Map<SnykFile, List<ScanIssue>>
+        issues: List<ScanIssue>,
+        securityIssuesCount: Int? = null,
+        fixableIssuesCount: Int? = null,
     ) {
-        fun navigateToSource(virtualFile: VirtualFile, textRange: TextRange): () -> Unit = {
-            io.snyk.plugin.navigateToSource(project, virtualFile, textRange.startOffset, textRange.endOffset)
+        // only add these info tree nodes to Snyk Code security vulnerabilities for now
+        if (securityIssuesCount == null) {
+            return
         }
+
+        val settings = pluginSettings()
+        // only add these when we enable the consistent ignores flow for now
+        if (!settings.isGlobalIgnoresFeatureEnabled) {
+            return
+        }
+
+        var text = "✅ Congrats! No vulnerabilities found!"
+        val issuesCount = issues.size
+        val ignoredIssuesCount = issues.count { it.isIgnored() }
+        if (issuesCount != 0) {
+            if (issuesCount == 1) {
+                text = "$issuesCount vulnerability found by Snyk"
+            } else {
+                text = "✋ $issuesCount vulnerabilities found by Snyk"
+            }
+            text += ", $ignoredIssuesCount ignored"
+        }
+        rootNode.add(
+            InfoTreeNode(
+                text,
+                project,
+            ),
+        )
+
+        if (fixableIssuesCount != null) {
+            if (fixableIssuesCount > 0) {
+                rootNode.add(
+                    InfoTreeNode(
+                        "⚡\uFE0F $fixableIssuesCount vulnerabilities can be fixed by Snyk DeepCode AI",
```

Comment:
Sure that that's the right emoji? Looks weird ⚡️.

Reply:
That's how my laptop copied over that emoji but I see in VSCode it's just `⚡` and not `⚡\uFE0F`. I'll change it to be like in VSCode